src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_ev_driver.cc

```diff
@@ -314,6 +331,10 @@ void grpc_ares_ev_driver_start_locked(grpc_ares_ev_driver* ev_driver) {
   if (!ev_driver->working) {
     ev_driver->working = true;
     grpc_ares_notify_on_event_locked(ev_driver);
+    grpc_millis timeout = 10000 + grpc_core::ExecCtx::Get()->Now();
```

Comment:
Please make this value configurable via a channel arg.

Reply:
Done. Added a test using it in recent change.